.github/CODEOWNERS

```diff
@@ -147,6 +147,9 @@
 /sendmail/                               @dabcoder david.bouchare@datadoghq.com
 /signl4/                                 @rons4 ron@signl4.com @DataDog/ecosystems-review
 /sigsci/                                 @signalsciences info@signalsciences.com
+/shipyard/                               @shipyard support@shipyard.build 
+/shoreline/                              @ivang-shoreline @ciprian-shoreline @alinshoreline @cioc-shoreline @cristi-shoreline support@shoreline.io @DataDog/marketplace-review
```

Comment:
Looks like this PR introduces changes to the CODEOWNERS of other integration folders. I assume this isn't intentional?

Reply:
Thanks for pointing this out -- I do not recall editing this file and assume it is unintentional?

Reply:
Fixed this!